### ISSUE TITLE
BuildCollectionList only returns one index in the TableList for tables with multiple indexes

### ISSUE THREAD

**Issue by denissun** [newrelic/nri-postgresql#94]
## Description
For tables with multiple indexes only one index appears in the TableList

## Expected Behavior
For tables with multiple indexes,  in the  TableList all of the indexes should be presented

## [Troubleshooting](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) or [NR Diag](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) results
Provide any other relevant log data.
TIP:  Scrub logs and diagnostic information for sensitive information 

## Steps to Reproduce

In src/collection/collection.go, line 170
current code:

```
170		if _, ok := schemaList[row.TableName.String]; !ok {
171			schemaList[row.SchemaName.String][row.TableName.String] = make([]string, 0)
172     }
```

New code

```
170		if _, ok := schemaList[row.SchemaName.String][row.TableName.String]; !ok {
171			schemaList[row.SchemaName.String][row.TableName.String] = make([]string, 0)
172     }
```

Using the current code, only one index appears, with the new code  a list of indexes present.

## For Maintainers Only or Hero Triaging this bug
*Suggested Priority (P1,P2,P3,P4,P5):*
*Suggested T-Shirt size (S, M, L, XL, Unknown):*

**Comment by alvarocabanas** [comment]
Hello, thanks for raising this issue, it has been accepted and it is going to be prioritized.

**Comment by carlossscastro** [comment]
@denissun 

We have released nri-postgres v2.10.5 with a couple of fixes including yours.

Thanks very much for your contribution
